### ISSUE TITLE
vgg weights

### ISSUE THREAD

**Issue by soans1994** [nikhitmago/semantic-segmentation-transfer-learning#1]
hello author,

your implementation uses vgg as feature extractor right? or does it transfer learn all parameters. if we use this and unfreeze layers, can we improve accuracy?
for param in vgg16.features.parameters():
  param.requires_grad = False

thank you
